tests/properties.yml

```diff
@@ -0,0 +1,24 @@
+master1:
```

Comment:
If we use tags instead hostname (see comment below) we could do dry here and just have:
```
+master_tag:
+  :roles:
+    - dnsmasq
+    - mesos_master
+    - marathon
+    - zookeeper
+slave_tag:
+  :roles:
+    - dnsmasq
+    - mesos_slave
+    - weave
+    - docker
```